docs/compile-xtrabackup.md

```diff
@@ -94,28 +91,33 @@ directory:
     $ cd build
     ```
 
-3. Run cmake or cmake3. In either case, the options you need to use are the
+3. If you use a regular build, run cmake or cmake3. In either case, the options you need to use are the
 same.
 
 !!! note
  
     You can build *Percona XtraBackup* with man pages but this requires
     `python-sphinx` package which isn’t available from that main repositories
     for every distribution. If you installed the `python-sphinx` package you
-    need to remove the `-DWITH_MAN_PAGES=OFF` from previous command.
+    need to remove the `-DWITH_MAN_PAGES=OFF` from the following command.
+
+```{.bash data-prompt="$"}
+$ cmake -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF ..
+```
+
+If you use [Pro builds](./pxb-pro.md), run the following command
 
 ```{.bash data-prompt="$"}
-$ cmake -DWITH_BOOST=PATH-TO-BOOST-LIBRARY -DDOWNLOAD_BOOST=ON \
--DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF -B ..
+$ cmake -DPROBUILD=1 -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF ..
 ```
 
 ### Parameter Information
 
 | **Parameter** | **Description** |
 |---------------|-----------------|
-| `-DWITH_BOOST` | For the `-DWITH_BOOST` parameter, specify the name of a directory to download the boost library to. This directory is created automatically in your current directory. |
+| `-DBUILD_CONFIG` | This option sets a specific build configuration, for example, `xtrabackup_release`. |
```

Comment:
This options builds a release/optimized version of the xtrabackup binary.

Reply:
I meant to say xtrabackup_release build release/optimized version of the binary

Reply:
updated